helm_deploy/prison-offender-events/templates/housekeeping-cronjob.yaml

```diff
@@ -20,5 +20,5 @@ spec:
               args:
                 - /bin/sh
                 - -c
-                - curl -XPUT http://prison-offender-events/queue-admin/retry-all-dlqs
```

Comment:
Actually I think the original is correct, it is the kubernettes service and, which is "prison-offender-events"